samples/bluetooth/mesh/light_ctrl/src/lc_pwm_led.c

```diff
@@ -23,11 +23,11 @@
 #endif
 
 #if DT_NODE_HAS_STATUS(PWM_LED0_NODE, okay)
-#define PWM_LABEL	DT_PWMS_LABEL(PWM_LED0_NODE)
+#define PWM_CTLR	DT_PWMS_CTLR(PWM_LED0_NODE)
 #define PWM_CHANNEL	DT_PWMS_CHANNEL(PWM_LED0_NODE)
 #else
 #error "Unsupported board: pwm-led0 devicetree alias is not defined"
-#define PWM_LABEL	""
+#define PWM_CTLR	""
```

Comment:
You can use `DT_INVALID_NODE` instead of using an empty string as a node identifier. You could also just remove the PWM_CTLR define altogether here. The below `DEVICE_DT_GET` would fail at link time, but the program will never link because of the `#error` on line 29, so it's not an issue. Your call.

Reply:
Fixed.